fury/shaders/base.py

```diff
@@ -126,6 +127,7 @@ def load_shader(shader_file):
     return load_text(shader_file)
 
 
+'''
 def load(filename):
```

Comment:
you should remove the quote and use the decorator instead:

```python
@deprecate_with_version(since='0.8.1', until='0.9.0',
                        message='Load function has been replaced by import_fury_shader.')
def load(filename):
    ....
```

Reply:
Done.